src/main/java/org/fundacionjala/coding/ketty/FizzBuzz.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.coding.ketty;
+
+import java.util.StringJoiner;
+
+/**
+ * @author ketty camacho.
+ * class fizzbuzz.
+ */
+public class FizzBuzz {
+    private static final int THREE = 3;
+    private static final int FIVE = 5;
+
+    /**
+     * @param number list of Fizz Buzz limit number.
+     * @return string of number.
+     */
+
+    public String fizzBuzzMethod(int number) {
```

Comment:
You can split this method in two
one of them given one number will return fizz, buzz, or fizz buzz